sonaranalyzer-dotnet/src/SonarAnalyzer.RuleDescriptorGenerator/SonarAnalyzer.RuleDescriptorGenerator.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
     <LangVersion>8.0</LangVersion>
     <AssemblyName>SonarAnalyzer.RuleDescriptorGenerator</AssemblyName>
     <RootNamespace>SonarAnalyzer.RuleDescriptorGenerator</RootNamespace>
     <OutputType>Exe</OutputType>
     <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
     <RestoreLockedMode>true</RestoreLockedMode>
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+    <TargetFrameworks>net48;netcoreapp3.1</TargetFrameworks>

```

Comment:
We need to target both .Net Framework and .Net Core because it is referenced by tests which are also targeting .Net Framework and .Net Core. Since it's a console app we cannot target .Net Standard.

In the long term, I think it would be better to split the test project we have now and add a dedicated test project for `RuleDescriptorGenerator`.

Reply:
well, I guess those tests just test the behavior of this class, right?

> In the long term, I think it would be better to split the test project we have now and add a dedicated test project for RuleDescriptorGenerator.

+1